julia/oxygen/Manifest.toml

```diff
@@ -0,0 +1,2 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Is this file really needed ?

Disclaimer: it is present for merly, but not aware of Julia usages

Reply:
Yup, we shouldn't need this file

Reply:
[5fae3de](https://github.com/the-benchmarker/web-frameworks/pull/7286/commits/5fae3de3237836c57e57c9822b1b1e3b59d46993)